basic/src/test/java/org/jboss/hal/testsuite/test/configuration/connector/MailTestCase.java

```diff
@@ -108,8 +110,10 @@ public void before() {
     }
 
     @AfterClass
-    public static void cleanUp() throws IOException, OperationException, InterruptedException, TimeoutException {
+    public static void cleanUp() throws IOException, OperationException, InterruptedException, TimeoutException, CommandFailedException {
         try {
+            client.apply(new RemoveSocketBinding(socketBinding));
+            client.apply(new RemoveSocketBinding(SOCKET_BINDING));
```

Comment:
That's exactly how our internal `RemoveSocketBinding` command behaves. https://github.com/wildfly-extras/creaper/blob/master/commands/src/main/java/org/wildfly/extras/creaper/commands/socketbindings/RemoveSocketBinding.java is going to be changed to be configurable to behave this way as well ... see https://github.com/wildfly-extras/creaper/issues/118